app/Http/Controllers/Api/V1/CurrikiGo/LmsController.php

```diff
@@ -81,43 +81,13 @@ public function activities(Request $request)
             'level' => 'string|max:255',
             'start' => 'string|max:255',
             'end' => 'string|max:255',
+            'author' => 'string|max:255',
+            'private' => 'integer',
+            'ltiClientId' => 'integer|required',
```

Comment:
We could implement Request validation for this. But it can come later I suppose.

Reply:
parking this for now.